internal/policy/verify.go

```diff
@@ -129,6 +140,93 @@ func VerifyRelativeForRef(ctx context.Context, repo *git.Repository, initialPoli
 	return nil
 }
 
+// VerifyCommit verifies the signature on the specified commits (identified by
+// their hash or via a reference that is resolved). For each commit, the policy
+// applicable when the commit was first recorded (directly or indirectly) in the
+// RSL is used.
+func VerifyCommit(ctx context.Context, repo *git.Repository, ids []string) map[string]string {
```

Comment:
Document what the return type is and how to interpret it.

e.g. it would be invalid if a commit isn't captured in the status. There also isn't a good way to separate out valid statuses from invalid statuses without inspecting the string (we may want to split these into valid/invalid statuses).

Reply:
I've updated the docstring to indicate how the status is to be parsed. Right now it's meant only for human consumption when they explicitly just use `verify-commit`. In the rest of the gittuf workflow, we don't currently want / need this. I do agree, though, that it may be worth breaking this up to return program-consumable statuses, with messages created at the CLI or repo package when it's for human consumption. Shall I track that as a ticket?